hr_holidays_leave_auto_approve/tests/test_hr_holidays_leave_auto_approve.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2016-2019 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo.tests.common import TransactionCase
+
+
+class TestHolidaysAutoValidate(TransactionCase):
+    def setUp(self):
```

Comment:
It'd be better to have this on `setUpClass` instead, to speed up tests.
Non blocking though